Apps/Common/src/AspNetConfiguration/Modules/ExceptionHandling.cs

```diff
@@ -37,26 +31,11 @@ public static class ExceptionHandling
         /// Adds and configures the services required to use problem details.
         /// </summary>
         /// <param name="services">The service collection to add forward proxies into.</param>
-        /// <param name="environment">The environment the services are associated with.</param>
-        public static void ConfigureProblemDetails(IServiceCollection services, IWebHostEnvironment environment)
+        public static void ConfigureProblemDetails(IServiceCollection services)
         {
-            services.AddProblemDetails(
-                setup =>
-                {
-                    setup.IncludeExceptionDetails = (_, _) => environment.IsDevelopment();
-
-                    setup.Map<ProblemDetailsException>(
-                        exception => new ProblemDetails
-                        {
-                            Title = exception.ProblemDetails?.Title,
-                            Detail = exception.ProblemDetails?.Detail,
-                            Status = (int)(exception.ProblemDetails?.StatusCode ?? HttpStatusCode.InternalServerError),
-                            Type = exception.ProblemDetails?.ProblemType,
-                            Instance = exception.ProblemDetails?.Instance,
-                        });
-
-                    setup.MapToStatusCode<CommunicationException>(StatusCodes.Status502BadGateway);
```

Comment:
This behaviour wasn't carried over to the new implementation.

Reply:
This should be resolved in the latest commit